src/components/navbar.js

```diff
@@ -33,7 +33,7 @@ function Navbar(props) {
             }}
           >
             <span
-              className={`fadeInUp ${view === "Home" ? "focused" : ""}`}
+              className={`fadeInUp ${props.location.pathname === "/" ? "focused" : ""}`}
               style={{ animationDelay: "0.2s" }}
```

Comment:
This can be made into a method. It will cutdown the verbosity.

```
const navProps = (path, animationDelay) =>  ({
   className: `fadeInUp ${props.location.pathname === path ? "focussed": ""}`,
   state={{ animationDelay: `${animationDelay}s`}}
})
```
```
<span {...navProps('/', 0.2)} />
  Home
</span>
```

Reply:
@vramana Thanks for your review. Working on it

Reply:
@vramana Please review latest changes